sale_exception_price_security/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================================
+Sale Exception - Price Security Integration
+===========================================
+
+When using together price_security and sale_exceptions, it changes the
```

Comment:
REEMPLAZAR:  
This module replaces the exceptions raised by price_security module for exceptions of the "Sale Exception" module. This gives better usability if you use Sale Exception module.
For more information on how to use please refer to "price_security" and "sale_exception" modules.